core/src/test/java/org/elasticsearch/cluster/routing/UnassignedInfoTests.java

```diff
@@ -69,6 +69,7 @@ public void testReasonOrdinalOrder() {
                 UnassignedInfo.Reason.REINITIALIZED,
                 UnassignedInfo.Reason.REALLOCATED_REPLICA,
                 UnassignedInfo.Reason.PRIMARY_FAILED,
+                UnassignedInfo.Reason.MANUAL_ALLOCATION,
```

Comment:
This should go at the end - what we test here is that we don't change the wire level byte format.